src/containers/PrimeStatusBanner/PrimeTokensLeft/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTranslation } from 'translation';
+
+const PrimeTokensLeft = ({ tokensLeft = 13 }: { tokensLeft: number }) => {
```

Comment:
I think you forgot to remove the default value:
```suggestion
const PrimeTokensLeft = ({ tokensLeft }: { tokensLeft: number }) => {
```

Also, to stay consistent of the codebase I'd recommend declaring the component prop types through an interface that can then be exported.